test/nn/test_encoding.py

```diff
@@ -20,3 +24,35 @@ def test_temporal_encoding(device):
 
     x = torch.tensor([1.0, 2.0, 3.0], device=device)
     assert encoder(x).size() == (3, 64)
+
+
+@withCUDA
+def test_link_encoding(device):
+    num_nodes = 3
+    num_edges = 6
+    num_edge_features = 10
+    edge_attr = torch.rand((num_edges, num_edge_features), device=device)
+    node_batch = torch.randint(low=0, high=num_nodes, size=(num_edges, ),
+                               device=device)
+    edge_time = torch.rand(num_edges, device=device)
```

Comment:
This is just for reference but according to the paper, the magnitude of `edge_time` is in the order of millions. In the testing, you are sampling from normal distribution. Might be good to align that in the testing code, or at least leave some comments.
<img width="746" alt="Screenshot 2023-06-02 at 6 47 14 PM" src="https://github.com/pyg-team/pytorch_geometric/assets/16249234/2bc30537-7cde-4659-95d0-be37b236b130">

Reply:
Do you mean we should use a large tensor within this unit test?

Reply:
I think either way is fine in terms of unit-test, but I think it's better to leave some note so that end-users are aware of the scale of the timestamps.

Reply:
Documented in the `LinkEncoding` module!

Reply:
Thank you!